postgres/9.6/Dockerfile

```diff
@@ -26,6 +26,11 @@ RUN git clone https://github.com/debezium/postgres-decoderbufs -b $PLUGIN_VERSIO
     && cd / \ 
     && rm -rf postgres-decoderbufs            
 
+RUN git clone https://github.com/eulerto/wal2json -b master --single-branch \
```

Comment:
Building from master makes me a bit wary due to the implicit instability. How about checking out a specific commit and getting in touch with wal2jon folks whether they can come up with a more table or tag at least?